### PR TITLE
release-master-2024-07-04-tagging-tool-improvements

### PR DESCRIPTION
**Pull request by sinatragianpaolo**

### CODE REVIEW

File: src/pages/Video/Actions.tsx [hunk 1]
```diff
@@ -91,21 +90,18 @@ const Actions = () => {
...
-                refScroll={refScroll}
-                key={observation.id}
-                observation={observation}
-                {...(video.transcript && {
-                  transcript: video.transcript,
-                })}
-              />
-            ))}
-        </div>
+        observations &&
```
Comment (sinatragianpaolo) [inline src/pages/Video/Actions.tsx:94]: questo non è necessario. Da rivedere

File: src/pages/Videos/parts/InfoRow.tsx [hunk 2]
```diff
@@ -1,45 +1,48 @@
-import { MD, SM } from '@appquality/unguess-design-system';
+import { Editor, MD, SM, Span } from '@appquality/unguess-design-system';
```
Comment (sinatragianpaolo) [inline src/pages/Videos/parts/InfoRow.tsx:1]: editor is defined but never used

File: src/pages/Videos/parts/InfoRow.tsx [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
   padding: ${(p) => p.theme.space.xs} 0;
-  column-gap: ${(p) => p.theme.space.xs};
-  ${StyledSM} {
-    flex-shrink: 0;
+`;
+
+const EditorContainer = styled.div`
```
Comment (sinatragianpaolo) [inline src/pages/Videos/parts/InfoRow.tsx:15]: editor container is not used

**Review by iDome89 — APPROVED** [review]

**Review by cannarocks — APPROVED** [review]
CannavacciuoloPazziGIF

**Pull request merged**